packages/toolkit/src/view/model/view-model/ModelRunViews/List.tsx

```diff
@@ -0,0 +1,253 @@
+import type { Model } from "instill-sdk";
+import * as React from "react";
+import Link from "next/link";
+
+import {
+  ColumnDef,
+  DataTable,
+  PaginationState,
+} from "@instill-ai/design-system";
+
+import {
+  RunsTableSortableColHeader,
+  RunStateLabel,
+} from "../../../../components";
+import {
+  convertToSecondsAndMilliseconds,
+  InstillStore,
+  ModelRun,
+  useInstillStore,
+  usePaginatedModelRuns,
+  useRouteInfo,
+  useShallow,
+} from "../../../../lib";
+import { getHumanReadableStringFromTime } from "../../../../server";
+import { TABLE_PAGE_SIZE } from "../constants";
+
+const selector = (store: InstillStore) => ({
+  accessToken: store.accessToken,
+  enabledQuery: store.enabledQuery,
+});
+
+export type ModelRunListProps = {
+  model?: Model;
+};
+
+const OWNER = {
+  id: null,
+};
+
+export const ModelRunList = ({ model }: ModelRunListProps) => {
+  const [orderBy, setOrderBy] = React.useState<string>();
+  const [paginationState, setPaginationState] = React.useState<PaginationState>(
+    {
+      pageIndex: 0,
+      pageSize: TABLE_PAGE_SIZE,
+    },
+  );
+  const { accessToken, enabledQuery } = useInstillStore(useShallow(selector));
+  const routeInfo = useRouteInfo();
+  const modelRuns = usePaginatedModelRuns({
+    accessToken,
+    enabled: enabledQuery && routeInfo.isSuccess,
+    modelName: `namespaces/${routeInfo.data.namespaceId}/models/${routeInfo.data.resourceId}`,
+    pageSize: TABLE_PAGE_SIZE,
+    page: paginationState.pageIndex,
+    orderBy,
+  });
+
+  const owner = React.useMemo(() => {
+    if (!model) {
+      return OWNER;
+    }
+
+    const owner = model.owner?.user || model.owner?.organization;
+
+    if (!owner || !owner.profile) {
```

Comment:
Since you didn't use profile here, I think we can safely remove this guard? I feel that under some condition, profile object may not be presented